.travis.yml

```diff
@@ -4,6 +4,13 @@ env:
   - version=1.14
   - version=1.15
 
+before_script:
+  - go get -u github.com/alecthomas/gometalinter
+  - go get -u github.com/go-ozzo/ozzo-validation
+  - go get -u github.com/go-ozzo/ozzo-validation/is
+  - gometalinter --install
+  - $HOME/gopath/src/github.com/sendgrid/sendgrid-go/prism.sh
```

Comment:
This is [no longer needed](https://github.com/sendgrid/sendgrid-go/blob/4d0ad813702fe8024b9111b6ab242f7126e77a4f/Makefile#L14).